src/browserExt/background.js

```diff
@@ -366,8 +385,16 @@ Zotero.Connector_Browser = new function() {
 		});
 	}
 	
-	function _save(tab) {
-		if(_tabInfo[tab.id] && _tabInfo[tab.id].translators && _tabInfo[tab.id].translators.length) {
+	function _browserAction(tab) {
+		if (Zotero.Prefs.get('firstUseNoClient')) {
+			Zotero.Prefs.set('firstUseNoClient', false);
+			if (Zotero.isFirefox && Zotero.Prefs.get('previousConnectorVersion') < '5') {
+				Zotero.Connector_Browser.openTab('https://www.zotero.org/support/zotero_for_firefox_migration');
+			} else {
+				Zotero.Connector_Browser.openTab('https://www.zotero.org/start_connector');
```

Comment:
Can use `/connector_start` for everything. We don't want to show new Firefox users a page that doesn't apply to them. We can detect Firefox on that page and show a section at the top with a link to another page.